src/flattenizer.ts

```diff
@@ -4,6 +4,18 @@
  * @module Flattenizer
  */
 
+export type Nullable<A> = A | null | undefined;
+export type Delimiter = string;
```

Comment:
💡 Noting for posterity that [TypeScript has documentation](https://www.typescriptlang.org/docs/handbook/advanced-types.html#type-aliases) about type aliases and how it's supported enough to not have additional overhead. At the moment I identify with the "not too terribly useful" comment the docs use- it is another abstraction and line of code for you to maintain, but you already have it and this is more stylistic, so no changes suggested.